BrainPortal/app/views/custom_filters/_new_task_custom_filter.html.erb

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
In all the view code for new custom filters and for editing custom filters, please use the attribute getters and setters that were generated with metaprogramming: that was the entire point of creating them, so that the form helpers use that as if they were standard attributes of the CustomFilter model.